frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -610,18 +616,77 @@ const ManagePolicyPage = ({
   }
 
   const renderPoliciesCount = (count?: number) => {
+    // Show count if there is no errors AND there are policy results or a search filter
+    const showCount =
+      count !== undefined &&
+      !policiesErrors &&
+      (policyResults || searchQuery !== "");
+
     return (
       <div className={`${baseClass}__count`}>
-        {count !== undefined && (
+        {showCount && (
           <span>{`${count} polic${count === 1 ? "y" : "ies"}`}</span>
         )}
       </div>
     );
   };
 
-  return !isRouteOk || (isPremiumTier && !userTeams) ? (
-    <Spinner />
```

Comment:
Root issue: can't use a spinner to replace the table level or else it will re-render (and thus unfocus) the search input field